### PR TITLE
#9706: Updated doc for case-sensitive filter

### PR DESCRIPTION
**Pull request by dylangrech92**
Updated the docs to show how to use case-sensitive filters

### CODE REVIEW

File: docs/en/02_Developer_Guides/00_Model/06_SearchFilters.md [hunk 1]
```diff
@@ -40,6 +41,22 @@ These suffixes can also take modifiers themselves. The modifiers currently suppo
...
+$players = Player::get()->filter([
+    'FirstName:ExactMatch' => 'Sam'
+]);
+
+// Fetch players that their FirstName is 'Sam' (case-sensitive)
+$players = Player::get()->filter([
+    'FirstName:ExactMatch:case' => 'Sam'
+]);
+
+// Shorthand versions of the above:
```
Comment (sminnee) [inline docs/en/02_Developer_Guides/00_Model/06_SearchFilters.md:55]: Maybe make it explicit that if no filter is supplied then ExactMatch is used as a default

File: docs/en/02_Developer_Guides/00_Model/06_SearchFilters.md [hunk 2]
```diff
@@ -40,6 +41,22 @@ These suffixes can also take modifiers themselves. The modifiers currently suppo
...
+$players = Player::get()->filter('FirstName', 'Sam'); // NOT case-sensitive
```
Comment (sminnee) [inline docs/en/02_Developer_Guides/00_Model/06_SearchFilters.md:56]: There is "nocase" as well, which ensures and insensitive match on all database backends (see the docs above)

**Review by sminnee — CHANGES_REQUESTED** [review]
A few small suggestions

**Comment by dylangrech92** [comment]
Both points are very valid, updated as suggested :)

**Review by sminnee — APPROVED** [review]
Thanks!

**Pull request merged**